samples/system-test/files.test.js

```diff
@@ -250,6 +250,21 @@ it('should get metadata for a file', () => {
   assert.match(output, new RegExp(`Bucket: ${bucketName}`));
 });
 
+it('should set metadata for a file', () => {
+  // set in sample
+  const userMetadata = {
+    my: 'custom',
+    properties: 'go here',
+  };
+  const output = execSync(
+    `node fileSetMetadata.js ${bucketName} ${copiedFileName}`
+  );
+  const metadata = eval(`(${output})`);
```

Comment:
I don't love the use of `eval` here, could we just use `JSON.parse()`, or `assert.match`?

Reply:
done